docs/source/guide/guide-executors.rst

```diff
@@ -423,3 +423,87 @@ Qiskit: Hardware
+  2. Unitary circuits only, so non-unitary errors need to use Monte Carlo
+
+Despite this latter limitation, there is a crossover point where Monte Carlo using Tensorflow evaluates faster than the exact density matrix simulation using ``circ``.
+
+Below is an example to use TensorFlow Quantum to simulate a bit-flip channel:
+
+.. code::
+
+    import numpy as np
+    from cirq import Circuit
+    import sympy
+
+    import tensorflow as tf
+    import tensorflow_quantum as tfq
+
+
+    def stochastic_bit_flip_simulation(circ: Circuit, p, num_MC=100):
+        """
+        Simulates a circuit with random bit flip (X(\pi)) errors
+        Args:
+            circ: The quantum program as a cirq object
+            p: probability of an X(\pi) gate on each qubit after each gate in circ
+            num_MC: number of random trajectories to average over
+        Returns:
+            The expectation value of the 0 state as a float
+        """
+    
+        #Create array of symbolic variables and reshape to natural circuit parameterization
+        h = sympy.symbols(''.join(['h_{0} '.format(i) for i in range(len(circ.moments)*len(circ.all_qubits()))]),positive=True)
+        h_array = np.asarray(h).reshape((len(circ.all_qubits()),len(circ.moments)))
+
+        #Symbolicly add X gates to the input circuit
+        noisy_circuit = Circuit()
+        for i,moment in enumerate(circ.moments):
+            noisy_circuit.append(moment)
+            for j,q in enumerate(circ.all_qubits()):
+                noisy_circuit.append(cirq.rx(h_array[j,i]).on(q))
+        
+        #rotations will be pi w/ prob p, 0 w/ prob 1-p
+        vals = [np.reshape((np.random.rand(len(circ.all_qubits()),len(circ.moments))<p)*np.pi,(1,len(circ.all_qubits())*len(circ.moments))) for _ in range(num_MC)]
+        
+
+        
+        # needs to be a rank 2 tensor
+        vals = np.squeeze(vals)
+        if num_MC == 1:
+            vals = [vals]
+        
+        #Instantiate tfq layer for computing state vector
+        state = tfq.layers.State()
+        
+        #Execute monte carlo sim with symbolic values specified by vals
+        out = state(noisy_circuit,symbol_names=h,symbol_values=vals).to_tensor()
+        
+        #Fancy way of computing and summing individual density operators, follwed by averaging
+        dm = tf.tensordot(tf.transpose(out),tf.math.conj(out),axes=[[1],[0]]).numpy()/num_MC
+        
+        #return measurement of 0 state
+        return np.real(dm[0,0])
+
+.. testcode::
+    :hide:
+    
+    #Issue with tfq prevents this working on build server, commenting out for now
+    
+    #import cirq
+    #from mitiq.benchmarks import randomized_benchmarking
+    #circ = randomized_benchmarking.rb_circuits(1,[20],1)[0]
+
+    ##Need to make sure the qubits are cirq.GridQubit
+    #circ=circ.transform_qubits(lambda q: cirq.GridQubit.rect(1,1)[0])
+
+    #out = stochastic_bit_flip_simulation(circ, .001)
+    #assert .97 < out < 1
```

Comment:
this fails sometimes. i've seen values as low as 0.93999755. so this range will have to be extended